src/Pmi/Entities/Participant.php

```diff
@@ -116,6 +117,12 @@ public function parseRdrParticipant($participant)
         if (!empty($participant->enrollmentStatus) && $participant->enrollmentStatus === 'FULL_PARTICIPANT') {
             $this->isCoreParticipant = true;
         }
+
+        // Set activity status
+        $this->activityStatus = $this->getActivityStatus($participant);
```

Comment:
I can't remember why we/I used this pattern of storing and accessing computed values in public variables. I don't think we should clean this up in this ticket/PR, but we should revisit at some point.